Node.gitignore

```diff
@@ -23,12 +27,12 @@ coverage
 # Compiled binary addons (http://nodejs.org/api/addons.html)
 build/Release
 
-# Dependency directories
-node_modules
-jspm_packages
+# Dependency directory
+*/node_modules
```

Comment:
I'm not clear on this change:

 - `node_modules` will ignore all folders with that name
 - `/node_modules` will ignore any folder at the root of the repository with this name

This change doesn't address either. Could you tell me more about what you're trying to define here?